src/common/utility/visit_gzstream.h

```diff
@@ -636,7 +636,11 @@ class UTILITY_API visit_ifstream {
    /** Simple factory to create ifstream or igzstream depending on whether
        named file appears to be a gzip'd compressed file.
        https://refspecs.linuxbase.org/LSB_3.0.0/LSB-PDA/LSB-PDA/zlib-gzopen-1.html. */
+#ifdef HAVE_LIBZ
```

Comment:
@biagas ... I recall exchanging emails with you recently on visit_gzstream+zlib. This PR apparently includes a fix to it.